Code/Geometry/point.h

```diff
@@ -70,24 +70,36 @@ class RDKIT_RDGEOMETRYLIB_EXPORT Point3D : public Point {
   inline unsigned int dimension() const override { return 3; }
 
   inline double operator[](unsigned int i) const override {
-    PRECONDITION(i < 3, "Invalid index on Point3D");
-    if (i == 0) {
-      return x;
-    } else if (i == 1) {
-      return y;
-    } else {
-      return z;
+    switch (i) {
+      case 0:
+        return x;
+        break;
```

Comment:
The `break`s here are not necessary since you are returning from the function. If you can remove the `break`s without having it introduce compiler warnings, it's probably worth doing so.